### PR TITLE
Document how to build a little bit better

### PR DESCRIPTION
**Pull request by Dinnerbone**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -1,4 +1,7 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
```
Comment (torokati44) [inline app/build.gradle.kts:1]: This `internal` here poked my eye a bit, but _I'm sure it's fine_.

File: README.md [hunk 2]
```diff
@@ -18,34 +18,9 @@ You can try this app by downloading and installing one of those.
...
-
-Opening the `app` project in Android Studio for development and debugging also works.
-
-# Development Tips
-
-Set the target of `rust-analyzer` to `x86_64-linux-android` (or even `aarch64-linux-android`), to stop one of the NDK crates from erroring out due to compilation to an unsupported platform.
-
-Also set the `ANDROID_HOME` and `ANDROID_NDK_ROOT` extra environment variables for it to suitable values, so the C parts of the dependencies can also be built by it.
-
-You may also need to set the `TARGET_CC`, `TARGET_CXX`, and `TARGET_AR` environment variables to the full paths of the (sometimes API level-specific) `clang`, `clang++`, and `llvm-ar` binaries, respectively.
```
Comment (torokati44) [inline README.md:48]: I'm trying to remember why I had to add this line... :thinking:

File: README.md [hunk 3]
```diff
@@ -18,34 +18,9 @@ You can try this app by downloading and installing one of those.
...
-The final APK should be at: `app/ruffle/build/outputs/apk/debug/ruffle-universal-debug.apk`
-There are also single-arch split APKs next to it.
-
-Opening the `app` project in Android Studio for development and debugging also works.
-
-# Development Tips
-
-Set the target of `rust-analyzer` to `x86_64-linux-android` (or even `aarch64-linux-android`), to stop one of the NDK crates from erroring out due to compilation to an unsupported platform.
-
-Also set the `ANDROID_HOME` and `ANDROID_NDK_ROOT` extra environment variables for it to suitable values, so the C parts of the dependencies can also be built by it.
```
Comment (torokati44) [inline README.md:46]: And perhaps these would still be useful for "the common folk" using VS Code...?

File: app/build.gradle.kts (hunk 1 above)
Comment (Dinnerbone) [reply app/build.gradle.kts:1]: It's the best we got... :D

File: README.md (hunk 2 above)
Comment (Dinnerbone) [reply README.md:48]: I kinda assumed it was pre-`cargo ndk`

File: README.md (hunk 3 above)
Comment (Dinnerbone) [reply README.md:46]: I'll add the `rust-analyzer` part back, but it already says about `ANDROID_NDK_ROOT`. I'm not sure if `ANDROID_HOME` is needed though? Maybe it is and I just didn't notice?

Comment (torokati44) [reply README.md:46]: `rust-analyzer`: Cool, thanks!
`ANDROID_HOME`: No idea, maybe it isn't (anymore?).

File: README.md (hunk 2 above)
Comment (torokati44) [reply README.md:48]: Maybe. Also, _maybe_ it had something to do with this...?
https://github.com/sonos/dinghy/issues/198#issuecomment-1832381916

**Comment by torokati44** [comment]
This is net positive, so merging. Thank you!
The other two hints about envvars, we can add back later if/when their reason pops up again.

**Pull request merged**